dashboard/app/views/parent_mailer/student_associated_with_parent_email.html.erb

```diff
@@ -0,0 +1,19 @@
+<p>Your child, <%= @student.name %>, has started learning Computer Science in their classroom at school on Code.org. We're thrilled to let you know that <%= @student.name %> is choosing to go beyond that and has just created a personal login to use even after the class is over. </p>
+
+<p>Your child can sign in at Code.org to learn any time with the username: <%= @student.username %> and their selected password. If they forget, choose “forgot password” and we’ll email you a password reset link. If you are not <%= @student.name %>’s parent, please reply to this email or contact support@code.org.</p>
+
+<h3>Looking for a place to get started?</h3>
+<p>At the top of their <a href="https://studio.code.org/home">homepage</a>, <%= @student.name %> can continue the course they are doing with their classroom at school. They can also create their own <a href="https://studio.code.org/projects/public">games or artwork in the Project Gallery</a>. Encourage them to browse the gallery and find something inspiring from another student their age. Challenge them to make their own code artwork and share it with you. You can print it and hang it up, email it to a relative, or share it on social media.</p>
+
+<h3>What is Code.org?</h3>
+<p>Code.org is a non-profit I started in 2013 with the vision that every student in every school should have the opportunity to learn computer science. Since then, our coding platform and computer science curriculum for grades K-12 have become the most broadly used in the world and reaches over 20% of all U.S. students. </p>
+
+<p>Your child's education is supported by generous donors that make Code.org free for every student. If your child enjoys Code.org, <a href="https://code.org/donate">please support us</a> so another child can learn for free too.</p>
+
+<p>Thanks,</p>
+<p>
+  Hadi Partovi<br/>
+  Founder, Code.org
+</p>
+
+<p>P.S. We take student privacy very seriously at Code.org. Read our <a href="http://code.org/privacy">privacy policy</a> for details.</p>
```

Comment:
No reason, was just following the other examples - I prefer haml as well, so I'm happy to switch it over